src/server_manager/web_app/digitalocean_server.ts

```diff
@@ -298,8 +299,8 @@ class DigitalOceanHost implements server.ManagedServerHost {
     return {usd: this.dropletInfo.size.price_monthly};
   }
 
-  getRegionId(): server.RegionId {
-    return this.dropletInfo.region.slug;
+  getCityName(): string {
```

Comment:
This is not returning a name, but a message id. Please update the name to reflect that.

But that really ties the model to localization, which is an application concept. It violates the layering. Domain model shouldn't depend on App. Ideally this should be a city id.

We can make up our own city ids, or reuse an existing one. I don't know of a good standard for city ids. However, google has its own standard that is used in [Analytics](https://developers.google.com/analytics/devguides/collection/protocol/v1/geoid), [Ads targeting](https://developers.google.com/adwords/api/docs/appendix/geotargeting) and [visualization tools](https://support.google.com/datastudio/answer/9843174?hl=en).

We could reuse that. The translation strings could be mapped from the city id (`city-${id}`).

Another possibility is to make the city id be `${region_code}-${normalize english name}`, which may be more readable. Region code would be using [ISO3166-2](https://en.wikipedia.org/wiki/ISO_3166-2).